tests/runner.py

```diff
@@ -249,6 +249,12 @@ def create_test_file(name, contents, binary=False):
   'wasm3',
   'wasms',
   'wasmz',
+  'wasm2js0',
```

Comment:
this, is kinda oddly named in the context of the test runner
because `wasm2` is a test suite
so `wasm2js3` kind of looks like "wasm with -O2, and js with -O3"

Don't think there's anything to do about that but here we are

Reply:
Yeah, the name is a little regrettable, but I don't have better ideas either...